pt-BR/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -5,7 +5,7 @@
 <!ENTITY fileMenu.accesskey                             "A">
 <!ENTITY newMenu.label                                  "Novo">
 <!ENTITY newMenu.accesskey                              "N">
-<!ENTITY newCard.label                                  "Address Book Card...">
+<!ENTITY newCard.label                                  "Cartão do catálogo de endereços...">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY newCard.label                                  "Cartão de visita...">